pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -653,10 +668,18 @@ func (c *Cluster) getOSDPropsForPVC(pvcName string) (osdProperties, error) {
 				logger.Infof("OSD will have its main bluestore block on %q", dataSource.ClaimName)
 			}
 
+			walSource, walOK := volumeSource.PVCSources[bluestorePVCWal]
+			if walOK {
+				logger.Infof("OSD will have its main bluestore block on %q, metadata device on %q and its wal device on %q", dataSource.ClaimName, metadataSource.ClaimName, walSource.ClaimName)
```

Comment:
Only need to log the wal device name since the rest was already logged on line 666